### ISSUE TITLE
Glovebox temperature readings nonsense

### ISSUE THREAD

**Issue by darrylmasson** [AG-Schumann/Doberman#34]
Probably due to a bad value or function in NTCtoTemp in labjack.py.

**Comment by darrylmasson** [comment]
For reference, the implementation in the old code:
```
# calculate resistance of 10kOhm NTC
R_NTC=Va/(((5.-Va)/10e3)-(8.181e-6*Va)+11.67e-6)/1000

# convert resistance to temperature
T_NTC=77.69-9.562*R_NTC+0.545*R_NTC**2-0.01183*R_NTC**3
```

**Comment by darrylmasson** [comment]
Because @ramirezdiego seems unwilling to investigate this, the problem appears to be the 4th parameter in the `T_NTC` conversion, which should be `0.01183` (as above) instead of the current value of `0.1183` (see [here](https://github.com/AG-Schumann/Doberman/blob/master/labjack.py#L25))

**Comment by darrylmasson** [comment]
Closed in the aftermath of #43
